From 4b4017b67fe436ab1fd835258d40fd11ee36317b Mon Sep 17 00:00:00 2001
From: PulkoMandy <pulkomandy@pulkomandy.tk>
Date: Sun, 2 Aug 2026 16:07:21 +0200
Subject: Remove Haiku special-case from os_types.h

This will default to config_types.h, which already defines the typedefs
in the standard way.

This avoids conflicting definitions if config_types.h is included first.

diff --git a/include/ogg/os_types.h b/include/ogg/os_types.h
index e655a1d..6c9cbba 100644
--- a/include/ogg/os_types.h
+++ b/include/ogg/os_types.h
@@ -78,17 +78,6 @@
    typedef int64_t ogg_int64_t;
    typedef u_int64_t ogg_uint64_t;
 
-#elif defined(__HAIKU__)
-
-  /* Haiku */
-#  include <sys/types.h>
-   typedef short ogg_int16_t;
-   typedef unsigned short ogg_uint16_t;
-   typedef int ogg_int32_t;
-   typedef unsigned int ogg_uint32_t;
-   typedef long long ogg_int64_t;
-   typedef unsigned long long ogg_uint64_t;
-
 #elif defined(__BEOS__)
 
    /* Be */
-- 
2.54.0

